ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -139,6 +139,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "courtBundle",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
+  },
```

Comment:
you could revert this file to master and it would match - i.e less change

Reply:
Is the courtBundle not needed here? Don't know why it wasn't there before but all the other document stuff is there.

Reply:
Without it you can't see the court bundle in the documents tab

Reply:
Wait a sec just realised what you mean...
It's Monday and in the morning is the excuse I'm going for

Reply:
I moved it there just so that it was grouped together with the other document permissions

Reply:
ok, fine to accept if it's more readable 👍